### PR TITLE
Release-Pivot

### PR DESCRIPTION
**Pull request by TraceLarson**
Added current project from WAI plus the following: 
Created Crime cards
Created modal map that shows location of crime and property location
Added Circular Progress bar for Family Friendly Component
Various design issues have been addressed.

Updated Documentation per DWA spec.

[DOCS:](https://github.com/TraceLarson/HouseFax/tree/dev/docs)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#[Brief](brief.md)
```
Comment (reactivepixel) [inline docs/README.md]: Best Practice: Your Markdown formatting is not valid.

> Change ```#[Brief](brief.md)``` into ```# [Brief](brief.md)```
> #[Brief](brief.md)
> # [Brief](brief.md)

File: routes/property.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		err ? console.log('Error creating property', err) : ''
+		res.send('Property Created')
+	})
+})
+
+
+router.put('/:id/likes', (req, res, next) => {
+	// From token
+	let token = {
+		userId: '5b933723b45f685e82d5212f'
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply routes/property.js]: I will update this issue as I progress with the Like button feature and the Profile page feature.

File: docs/spec.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            - CrimeMap Component
+            - CrimeCard Component
+            - Description Component
+            - AgentContactForm Component
+            - DetailsImage Component
+            - PropertyDetails Component
+            - RecentCrimeReports Component
+            
+        
+- **Sitemap**
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/spec.md]: Commit: 0fc1d16a143d1071336214058a243d563c533b4f
addresses this issue

File: docs/spec.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    - Home -> -> -> ->  Search -> Results Page -> Details Page -> Modal Map
+      |                ^
+      -> Login -> Home |
+      |                |
+      -> Register -> -> -> Profile
+    ```
+           
+- **Interface**
+    - _Information Architecture_
+        - HomePage Nav: HOUSE FAX Austin
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/spec.md]: Commit: a5561efa11962c25e3d88f442310b4452f30f27e
addresses this issue

File: docs/spec.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+#Spec
+- **Feature Definitions**
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/spec.md]: Commits: 
c40578e40aab5168d2992d860c10850da01e8836 
and 
1437434c3c39e693ece3fa6be452707e9ba3de77
address this issue.

File: docs/brief.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- **Audience**
+    - The Primary audience will be technology literate users between the ages of 25 to 40 years old and a HighSchool grad or above.
+    Users should be in a position that makes them ready to purchase a home or make a move to a new community. Ideal users would
+    be shopping for better family accommodations.
+- **Problem**
+    - The problem with the current real estate applications is that they put crime and the surrounding neighborhood secondary to every other statistic available.
+- **Solution**
+    - This application will show crime statistics first and foremost. Emphasis will be put on any available data starting from the most recent occurence.
+- **Unique Value Proposition**
+    - This application is separated by being light weight and instead of being feature rich, will aim to one or two things really well.
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/brief.md]: Commit: 2ae590a17363698e5d385ded6d0dbcb50d64d555
addresses this issue

File: docs/brief.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Brief
+- **Audience**
+    - The Primary audience will be technology literate users between the ages of 25 to 40 years old and a HighSchool grad or above.
+    Users should be in a position that makes them ready to purchase a home or make a move to a new community. Ideal users would
+    be shopping for better family accommodations.
+- **Problem**
+    - The problem with the current real estate applications is that they put crime and the surrounding neighborhood secondary to every other statistic available.
+- **Solution**
+    - This application will show crime statistics first and foremost. Emphasis will be put on any available data starting from the most recent occurence.
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/brief.md]: Commit: 2ae590a17363698e5d385ded6d0dbcb50d64d555
addresses this issue.

File: docs/brief.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+# Brief
+- **Audience**
+    - The Primary audience will be technology literate users between the ages of 25 to 40 years old and a HighSchool grad or above.
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/brief.md]: Commit: 79372a24d5d4d536c1c191d14774ccf3107c6b1a
addresses this issue

File: docs/brief.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+# Brief
```
[the remark this answers is not in the document]
Comment (TraceLarson) [reply docs/brief.md]: Commit: 315c21b83a3cd84b6192d55d85829fbb6cdf3c47
addresses this issue

**Comment by TraceLarson** [comment]
@reactivepixel Review of changes requested.

**Comment by reactivepixel** [comment]
Changes approved!

**Pull request merged**